tide/core/scene/ContentType.h

```diff
@@ -53,9 +53,17 @@ enum CONTENT_TYPE
     CONTENT_TYPE_TEXTURE,
     CONTENT_TYPE_PDF,
     CONTENT_TYPE_WEBBROWSER,
+    CONTENT_TYPE_WHITEBOARD,
     CONTENT_TYPE_IMAGE_PYRAMID
 };
 
+enum StreamType
+{
+    STANDARD,
```

Comment:
don't forget to rebase your branch on master, this enum was modified:
https://github.com/BlueBrain/Tide/pull/97/files#diff-c5d7e3f8e70fc822ebefde7ca74b268fR57